api/server/server.go

```diff
@@ -206,6 +226,9 @@ func WithAgent(agent agent.Agent) ServerOption {
 // New creates a new Functions server with the opts given. For convenience, users may
 // prefer to use NewFromEnv but New is more flexible if needed.
 func New(ctx context.Context, opts ...ServerOption) *Server {
+	span, ctx := opentracing.StartSpanFromContext(ctx, "server_init")
```

Comment:
this was to try to repro the span panic before the tracer is set up, which wasn't successful. seems useful though to leave, in case it takes forever to ping sql db or something in init (need to add sub spans in init for stuff like that)